doc/source/whatsnew/v0.20.0.txt

```diff
@@ -116,6 +116,7 @@ Other enhancements
 
 - ``.select_dtypes()`` now allows the string 'datetimetz' to generically select datetimes with tz (:issue:`14910`)
 
+- ``pd.TimedeltaIndex`` now has a custom datetick formatter specifically designed for nanosecond level precision (:issue:`8711`)
```

Comment:
That is certainly possible, we only have not really a section about timedelta plotting currently.